Source/Shaders/SkyAtmosphereFS.glsl

```diff
@@ -32,15 +32,38 @@
  
  // Code:  http://sponeil.net/
  // GPU Gems 2 Article:  http://http.developer.nvidia.com/GPUGems2/gpugems2_chapter16.html
- 
+ // HSV <-> RGB conversion with minimal branching: http://lolengine.net/blog/2013/07/27/rgb-to-hsv-in-glsl
+
+uniform vec3 u_hsvShift; // hue, saturation, value
+
 const float g = -0.95;
 const float g2 = g * g;
+const float epsilon = 1.0e-10;
```

Comment:
Given this is 32-bit floating-point, I don't think we can expect more than seven decimal digits of precision (roughly) so epsilon 7 should be enough.